Reason: Latest tagged range_v3 release does not work with cxx20 and
	and causes gcc to segfault

--- cmake/init_target.cmake	2020-01-21 21:30:05.000000000 +0100
+++ -	2020-01-22 01:29:33.006630878 +0100
@@ -15,11 +15,11 @@
         if (${ARGV1} STREQUAL cxx_std_14 OR ${ARGV1} STREQUAL cxx_std_11)
             target_compile_features(${target_name} PUBLIC ${ARGV1})
         else()
-            target_compile_features(${target_name} PUBLIC cxx_std_20)
+            target_compile_features(${target_name} PUBLIC cxx_std_17)
             init_target_folder(${target_name} ${ARGV1})
         endif()
     else()
-        target_compile_features(${target_name} PUBLIC cxx_std_20)
+        target_compile_features(${target_name} PUBLIC cxx_std_17)
     endif()
     if (WIN32)
         set_target_properties(${target_name} PROPERTIES
